R/pdf_document.R

```diff
@@ -109,6 +109,7 @@ pdf_document <- function(toc = FALSE,
   # template path and assets
   if (identical(template, "default")) {
 
+    if (!pandoc_available()) stop("Pandoc not found")
```

Comment:
From the perspective of a customer using Shinyapps.io, they will see "Pandoc not found" and likely open a support ticket asking if it is available, which will loop in both support and a developer.

Please consider making the error message as informative as is necessary for an end user to determine the nature of their mistake and correct it, without resorting to opening a support case.